common/src/main/kotlin/com/hartwig/actin/molecular/evidence/known/KnownEventResolverFactory.kt

```diff
@@ -13,24 +13,29 @@ import com.hartwig.serve.datamodel.molecular.range.KnownExon
 
 object KnownEventResolverFactory {
 
-    val KNOWN_EVENT_SOURCES = setOf(ActionabilityConstants.EVIDENCE_SOURCE)
+    val PRIMARY_KNOWN_EVENT_SOURCE = ActionabilityConstants.EVIDENCE_SOURCE
 
     fun create(knownEvents: KnownEvents): KnownEventResolver {
-        return KnownEventResolver(filterKnownEvents(knownEvents), GeneAggregator.aggregate(knownEvents.genes()))
+        val primaryKnownEvents = filterKnownEvents(knownEvents, filterPrimaryEvents = true)
+        val secondaryKnownEvents = filterKnownEvents(knownEvents, filterPrimaryEvents = false)
```

Comment:
For me "include" sounds like a better term than "filter" since filter implies you remove objects or you don't. In the implementation you _include_ objects yes or no